superset/viz.py

```diff
@@ -2062,6 +2062,40 @@ def get_data(self, df):
         }
 
 
+class DeckPolygon(BaseDeckGLViz):
+
+    """deck.gl's Polygon Layer"""
+
+    viz_type = 'deck_polygon'
+    verbose_name = _('Deck.gl - Polygon')
+    deser_map = {
+        'json': json.loads,
+        'polyline': polyline.decode,
+    }
+
+    def get_position(self, d):
+        return d[self.form_data.get('line_column')]
+
+    def query_obj(self):
+        d = super(DeckPolygon, self).query_obj()
```

Comment:
@mistercrunch not sure if you wanted me to move this functionality into the baseClass? I think this is fine since we are still leveraging the `line_column` controls. Let me know what you think.

Reply:
In theory we could have some intermediate (or mixin class) in between the base class that would be common to deck_path and deck_polygon, but in general the python philosophy is to nest inheritance simple as much as possible. I'm fine leaving it as is as it's little code duplication as it is.

Reply:
Now that I think about it, could we just have `DeckPolygon` derive `DeckPath`? Aren't they pretty much identical?